lib/src/mobile_scanner_controller.dart

```diff
@@ -58,9 +59,25 @@ class MobileScannerController {
   /// Automatically start the mobileScanner on initialization.
   final bool autoStart;
 
+  /// Can be used to override default Android camera resolution.
```

Comment:
Could you rename this property to just `cameraResolution` ? Maybe one day, we can use this to configure resolutions on other platforms as well.

```suggestion
/// The desired resolution for the camera.
///
/// When this value is provided, the camera will try to match this resolution,
/// or fallback to the closest resolution available.
/// When this is null, Android defaults to a resolution of 640x480.
///
/// Bear in mind that changing the resolution has an effect on the aspect ratio.
///
/// When the camera orientation changes,
/// the resolution will be flipped to match the new dimensions of the display.
///
/// Currently only supported on Android.
final Size? cameraResolution;
```